var/spack/repos/builtin/packages/verrou/package.py

```diff
@@ -28,7 +28,7 @@
 import sys
 
 
-class Verrou(AutotoolsPackage):
+class Verrou(AutotoolsPackage, PythonPackage):
```

Comment:
I want to say that this autotools-based package provides python extensions. Is multiple inheritance the right way to do it? It seems to work currently, but I wouldn't want to rely on implementation-specific behaviour...

Reply:
I'm pretty sure that is not the way to go. These base classes provide conflicting implementations for at least the `build` and `install` phases. 

If you only want to provide a python extension then you can add
```
extends('python')
```

But we should probably cover against this kind of implicit override... (Note: There is `CudaPackage` but it is not really a package in the sense of `AutotoolsPackage`).

Please correct me if I'm wrong @adamjstewart @alalazo

Reply:
Works for me. If there is no other comment, then I guess this PR is ready :smile: